### ISSUE TITLE
node recording gets crashed in AArch64 

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#791]
I tested tracing node in AArch64, but it gets segfault as follows:
```
$ file node
node: ELF 64-bit LSB executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, BuildID[sha1]=10b4c6f29ba4f89875e8ccd239c825fadcc0a273, with debug_info, not stripped

$ uftrace record --no-libcall ./node -e ''
child terminated by signal: 11: Segmentation fault
```
The node was cross-compiled as described in [uftrace for node](https://github.com/namhyung/uftrace/wiki/uftrace-for-node).

**Comment by honggyukim** [comment]
```
$ uftrace replay
# DURATION     TID     FUNCTION
            [  2093] | _GLOBAL__sub_I__register_cares_wrap() {
   2.135 us [  2093] |   uv_mutex_init();
  48.073 us [  2093] | } /* _GLOBAL__sub_I__register_cares_wrap */
            [  2093] | _GLOBAL__sub_I_node::reverted() {
            [  2093] |   node::MutexBase::MutexBase() {
  20.416 us [  2093] |     uv_mutex_init();
  22.708 us [  2093] |   } /* node::MutexBase::MutexBase */
  66.979 us [  2093] |   node::PerProcessOptions::PerProcessOptions();
            [  2093] |   node::MutexBase::MutexBase() {
   0.677 us [  2093] |     uv_mutex_init();
   2.292 us [  2093] |   } /* node::MutexBase::MutexBase */
 126.302 us [  2093] | } /* _GLOBAL__sub_I_node::reverted */
            [  2093] | _GLOBAL__sub_I_node::http2::Http2Session::callback_struct_saved() {
            [  2093] |   node::http2::Http2Session::Callbacks::Callbacks() {
   5.677 us [  2093] |     nghttp2_session_callbacks_new();
   0.468 us [  2093] |     nghttp2_session_callbacks_set_on_begin_headers_callback();
   0.365 us [  2093] |     nghttp2_session_callbacks_set_on_header_callback2();
   0.521 us [  2093] |     nghttp2_session_callbacks_set_on_frame_recv_callback();
...
   0.469 us [  2093] |                                     v8::internal::WeakFixedArray::Set();
   6.979 us [  2093] |                                   } /* v8::internal::DescriptorArray::CopyFrom */
 109.843 us [  2093] |                                 } /* v8::internal::DescriptorArray::CopyUpToAddAttributes */
            [  2093] |                                 v8::internal::DescriptorArray::Append() {
   0.417 us [  2093] |                                   v8::internal::WeakFixedArray::Set();

uftrace stopped tracing with remaining functions
================================================
task: 2093
[16] v8::internal::DescriptorArray::Append
[15] v8::internal::Map::CopyAddDescriptor
[14] v8::internal::Map::CopyWithField
[13] v8::internal::Map::TransitionToDataProperty
[12] v8::internal::LookupIterator::PrepareTransitionToDataProperty
[11] v8::internal::Object::AddDataProperty
[10] v8::internal::JSObject::AddProperty
[9] v8::internal::Genesis::InitializeGlobal_harmony_sharedarraybuffer
[8] v8::internal::Genesis::InitializeExperimentalGlobal
[7] v8::internal::Genesis::Genesis
[6] v8::internal::Bootstrapper::CreateEnvironment
[5] v8::NewContext
[4] v8::Context::New
[3] node::NewContext
[2] node::Start
[1] node::Start
[0] main

task: 2095
[1] uv_run
[0] node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start::$_0::_FUN

task: 2096
...
```

**Comment by honggyukim** [comment]
I just found that the `mcount_return` has to keep more registers and I will summary the problem and its fix code soon.

**Comment by honggyukim** [comment]
It is fixed with #792.
```
$ uftrace record ./node -e ''

$ uftrace info
# system information
# ==================
# program version     : v0.9.2-184-gc260 ( dwarf python tui perf sched dynamic )
# recorded on         : Sun Jun  9 19:17:46 2019
# cmdline             : uftrace record ./node -e 
# cpu info            : ARM64 (v8)
# number of cpus      : 4 / 4 (online / possible)
# memory info         : 184.6 / 918.6 MB (free / total)
# system load         : 1.05 / 0.67 / 0.38 (1 / 5 / 15 min)
# kernel version      : Linux 4.14.0-3-arm64
# hostname            : rpi3
# distro              : "Debian GNU/Linux buster/sid"
#
# process information
# ===================
# number of tasks     : 7
# task list           : 3579(node), 3581(node), 3582(node), 3583(node), 3584(node), 3585(node), 3586(node)
# exe image           : /home/honggyu/node/node
# build id            : 10b4c6f29ba4f89875e8ccd239c825fadcc0a273
# pattern             : regex
# exit status         : exited with code: 0
# elapsed time        : 13.362498371 sec
# cpu time            : 0.220 / 12.555 sec (sys / user)
# context switch      : 56 / 198 (voluntary / involuntary)
# max rss             : 45912 KB
# page fault          : 1 / 6914 (major / minor)
# disk iops           : 16 / 8 (read / write)
```
It successfully records more complicated node execution as follows:
```
$ uftrace record -a ./node --expose-gc --trace-gc -e 'console.log("Hello AArch64!"); gc();'
[3588:0x49bd81f0]     5297 ms: Scavenge 2.5 (4.2) -> 2.2 (5.2) MB, 135.5 / 0.4 ms  (average mu = 1.000, current mu = 1.000) allocation failure
[3588:0x49bd81f0]     7522 ms: Scavenge 2.7 (5.2) -> 2.6 (5.2) MB, 151.7 / 0.1 ms  (average mu = 1.000, current mu = 1.000) allocation failure
Hello AArch64!
[3588:0x49bd81f0]    14596 ms: Mark-sweep 3.6 (5.2) -> 3.2 (5.7) MB, 819.9 / 0.2 ms  (average mu = 1.000, current mu = 1.000) testing GC in old space requested
```
